### PR TITLE
Corrected typos, dates and versions in the index, setup and tips.

### PR DESCRIPTION
**Pull request by homberghp**

### CODE REVIEW

**Comment by homberghp** [comment]
Added week9, sealed shape project in topics/code as example and included in description.

File: topics/mavenmodules.adoc [hunk 1]
```diff
@@ -22,7 +22,13 @@ inter-dependencies. Otherwise it is just plain maven.
 .Snippet from a parent pom
 [source,xml]
 ----
-    <groupId>nl.fontys.sebivenlo</groupId>
+    <<parent>
+      <groupId>io.github.fontysvenlo</groupId>
+      <artifactId>informaticspom</artifactId>
+      <version>1.2</version>
```
Comment (ibrahimko) [inline topics/mavenmodules.adoc:28]: shouldn't be here 1.3?

File: topics/mavensettingsandrepo.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ A maven repository is a server that provides ready built maven 'artifacts', such
 
 There is a default repository, called https://mvnrepository.com/repos/central[maven central]. You (as a team or company) can have your own repository.
 
-If you look at a maven repository you will see that it is organized along the three axes: groupId, artifactId and version. We have published our Fontys Venlo *informatics pom* under groupId io.github.fontysvenlo, artifactId informaticspom and current (February 2023) 0.9. 
+If you look at a maven repository you will see that it is organized along the three axes: groupId, artifactId and version. We have published our Fontys Venlo *informatics pom* under groupId io.github.fontysvenlo, artifactId informaticspom and current (February 2024) 1.2.
```
Comment (ibrahimko) [inline topics/mavensettingsandrepo.adoc:12]: not version 1.3?

File: topics/code/sealedshapes/pom.xml [hunk 3]
```diff
@@ -2,18 +2,20 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>io.github.sebivenlo</groupId>
-        <artifactId>testeasypom</artifactId>
-        <version>5.2.0</version>
+        <groupId>io.github.fontysvenlo</groupId>
+        <artifactId>informaticspom</artifactId>
+        <version>1.2</version>
```
Comment (ibrahimko) [inline topics/code/sealedshapes/pom.xml]: again here 1.3?

**Review by ibrahimko — APPROVED** [review]

File: topics/mavenmodules.adoc (hunk 1 above)
Comment (homberghp) [reply topics/mavenmodules.adoc:28]: yes

File: topics/mavensettingsandrepo.adoc (hunk 2 above)
Comment (homberghp) [reply topics/mavensettingsandrepo.adoc:12]: yes

File: topics/code/sealedshapes/pom.xml (hunk 3 above)
Comment (homberghp) [reply topics/code/sealedshapes/pom.xml]: yes  ;-(

**Pull request merged**